clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/StreamsAvroExample.java

```diff
@@ -69,15 +69,18 @@ public static void main(String[] args) throws Exception {
         props.put(StreamsConfig.APPLICATION_ID_CONFIG, "demo-streams-avro-1");
         // Disable caching to print the aggregation value after each record
         props.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0);
-        props.put(StreamsConfig.REPLICATION_FACTOR_CONFIG, 3);
         props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
 
         final Serde<DataRecordAvro> dataRecordAvroSerde = new SpecificAvroSerde<>();
         final boolean isKeySerde = false;
         Map<String, Object> SRconfig = new HashMap<>();
         SRconfig.put(AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, props.getProperty(AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG));
-        SRconfig.put("schema.registry.basic.auth.user.info", props.getProperty("schema.registry.basic.auth.user.info"));
-        SRconfig.put("basic.auth.credentials.source", props.getProperty("basic.auth.credentials.source"));
+        if (props.getProperty("schema.registry.basic.auth.user.info") != null) {
+            SRconfig.put("schema.registry.basic.auth.user.info", props.getProperty("schema.registry.basic.auth.user.info"));
+        }
+        if (props.getProperty("basic.auth.credentials.source") != null) {
+            SRconfig.put("basic.auth.credentials.source", props.getProperty("basic.auth.credentials.source"));
+        }
```

Comment:
As these values are now deprecated, should the examples transition to using the post 5.0 supported values? https://docs.confluent.io/platform/current/schema-registry/security/index.html

Additionally, there are constant provided values in the AbstractKafkaSchemaSerDeConfig that can be used instead of the repetitive string values to improve compile time checking.

Reply:
@rspurgeon what do you think about the latest change in the latest commit?

Reply:
@ybyzek I think this simplified version also works when converting the Properties to a Map:

`Map<String, String> SRconfig = (Map)props;`

Reply:
Validated @rspurgeon thanks for the simplification